js/src/dapps/registry/events/events.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component, PropTypes } from 'react';
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+
+const { IdentityIcon } = window.parity.react;
```

Comment:
I would suggest to keep all possible "globals" in a single module and import it when needed:
```js
import { IdentityIcon } from '../parity';
```

And
```js
// parity.json

export window.parity.react.IdentityIcon;
```